core/src/main/scala/org/broadinstitute/dsde/rawls/billing/AzureBillingProjectLifecycle.scala

```diff
@@ -184,7 +183,7 @@ class AzureBillingProjectLifecycle(
     *  starts a landing zone deletion job
     *  does not ensure that the landing zone deletion completes successfully.
     */
-  private def cleanupLandingZone(
+  def cleanupLandingZone(
```

Comment:
Called by `BillingProjectOrchestrator` if the billing project has a `landingZoneId` (called one other place within `AzureBillingProjectLifecycle` as well).